package.json

```diff
@@ -1,24 +1,72 @@
 {
   "name": "p5",
+  "version": "2.0.3",
+  "description": "[![npm version](https://badge.fury.io/js/p5.svg)](https://www.npmjs.com/package/p5)",
+  "homepage": "https://p5js.org",
+  "bugs": {
+    "url": "https://github.com/processing/p5.js/issues"
+  },
   "repository": "processing/p5.js",
+  "license": "LGPL-2.1",
+  "author": "",
```

Comment:
As far as I understand, the "author" field is a best practice to include but optional. Because this project follows the All Contributors specification, I think it could either be omitted or using something like "p5.js contributors"? Any thoughts on this? Minor preference against adding this as a blank field.

Reply:
The author field here is not quite added but just rearranged from elsewhere in this file, ie. it is there already before as a blank string. I'm not sure this field is read by anything specific (that I know of) to populate something so I think something like "p5.js contributors" is fine either way.